common/functions-v2/src/scalars/string.rs

```diff
@@ -77,22 +98,31 @@ pub fn register(registry: &mut FunctionRegistry) {
     registry.register_aliases("char_length", &["character_length"]);
 }
 
+// Vectorize string to string function with default estimate_bytes, which is the original string length.
 fn vectorize_string_to_string(
```

Comment:
This can be achieved by `common_expression::function::vectorize_with_writer_1_arg`

Reply:
Oh sorry, I misunderstood it. But also, it may be not necessary to have both `vectorize_string_to_string` and `vectorize_string_to_string_wise`, a single `vectorize_string_to_string_wise` is enough to handle the case of same length output.

Reply:
Let's simple use `vectorize_string_to_string_wise`, and rename it to `vectorize_string_to_string`.